datasets/pgc-rema.yaml

```diff
@@ -1,4 +1,4 @@
-Name: REMA
+Name: REMA DEM
```

Comment:
@clairecporter The name needs to be at least 5 chars in length so added "DEM".

Reply:
I used the full dataset name instead.  Thanks for figuring that out.